go.mod

```diff
@@ -1,11 +1,12 @@
 module github.com/majewsky/schwift
 
-go 1.20
+go 1.21
```

Comment:
Please revert. I set this deliberately in https://github.com/majewsky/schwift/commit/17c1ae81f02a4b2a24e79d4018dfb1061962420d yesterday.

Reply:
Then we can't use go-bits and errext.As

Reply:
Then we should copy the respective functions over into this package as `internal`. It's only two small functions, should not be too much duplication.